### ISSUE TITLE
Cootamundra Gundagai Regional Council

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/issues#149]
This issue has been **automatically** created by PlanningAlerts for [Cootamundra Gundagai Regional Council ](http://www.planningalerts.org.au/authorities/cootamundra_gundagai)

It uses the scraper [planningalerts-scrapers/cootamundra](https://morph.io/planningalerts-scrapers/cootamundra)

Only close this issue once the authority is working again on PlanningAlerts. Otherwise a new issue will just automatically created. Also, if there is a duplicate issue close the other one in favour of this one.

**Comment by mlandauer** [comment]
It looks like they have a page for applications on exhibition https://www.cgrc.nsw.gov.au/development-applications-on-public-exhibition/ but it looks complete hand-made and not really worth trying to scrape. I can't find mention of DA Tracking or anything similar. I'm going to label this as "blocked by authority"

**Comment by jamezpolley** [comment]
Notes on page say that applications are now available in the NSW portal and not on the council site.

I've found two ways on the Planning Portal to search for information about applications:

- https://www.planningportal.nsw.gov.au/map which uses [the api](https://api.apps1.nsw.gov.au/eplanning/data/v0/DAApplicationTracker) to pull data:
- 
```shell
curl 'https://api.apps1.nsw.gov.au/eplanning/data/v0/DAApplicationTracker' \
  -H 'Origin: https://www.planningportal.nsw.gov.au' \
  -H 'Referer: https://www.planningportal.nsw.gov.au/' \
  --data-raw '{"PageNumber":1,"PageSize":4,"ApplicationStatus":"ALL","CouncilDisplayName":"Cootamundra-Gundagai Regional Council"}'
```

At first glance this looks good, but at second glance maybe not. It seems that it's returning something along the lines of a list of all applications in the system along with an indicator of whether or not they match my query:

        {
            "FullAddress": "7 HARMONY CLOSE BELLBIRD 2325",
            "IS_SELECTED": "N"
        },

The results that show up on the page don't show anything like a description for the DA, but at least for the ones currently on exhibition, the URL for the DA gives a hint: https://pp.planningportal.nsw.gov.au/daex/exhibition/electricity-generating-facility-construction-499-mw-ac-solar-farm-including-four-batteries. But that particular one appears to be exceptional - most don't show that much information.

There's another way to get [a list of applications on exhibition](https://www.planningportal.nsw.gov.au/daexhibitions): https://www.planningportal.nsw.gov.au/daexhibitions?field_daex_status_value=All&field_local_government_area_value=BERRIGAN&field_application_type_value=All&field_panel_reference_number_value=&field_council_unique_number_value=&title=&field_exhibition_start_date_value=&field_exhibition_end_date_value=

That page is restricted (it only shows applications that are on, or have been on, exhibition, rather than things like CDCs). It does show a little bit more information, but it's all information that is already on [the page for the DA](https://pp.planningportal.nsw.gov.au/daex/under-consideration/87-123-hamilton-street-finley-agricultural-produce-industry)

In short - it looks like there's two routes to getting information about applications that are going to be on exhibition - the API and the daexhibitions form. The latter gives slightly more information on the page, but both link through to details page about the DA in question which has the full set of detail That page doesn't reliably have a useful description either. Even applications currently on display don't have a description.

Or  to put it even more simply - see [Kat's comment below](https://github.com/planningalerts-scrapers/issues/issues/149#issuecomment-1786277117)

**Comment by katska** [comment]
DAs only available on NSW Planning Portal, which doesn't give a description.

**Comment by ianheggie-oaf** [comment]
CONFIRMED label: "nsw planning portal" since it is NOT listed in [Council application trackers](https://www.planningportal.nsw.gov.au/map/council-trackers)
